src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -51,28 +51,52 @@ module SoilBiogeochemDecompCascadeBGCMod
   integer, private, parameter :: i_litr1   = i_met_lit  ! First litter pool, metobolic
   integer, private, parameter :: i_litr2   = i_cel_lit  ! Second litter pool, cellulose
   integer, private, parameter :: i_litr3   = i_lig_lit  ! Third litter pool, lignin
+  integer, private, parameter :: i_micr1   = i_micr_r   ! First microbial pool, copiotrophic, as labeled in Wieder et al. 2015
+  integer, private, parameter :: i_micr2   = i_micr_k   ! Second microbial pool, oligotrophic, as labeled in Wieder et al. 2015
 
```

Comment:
were are the som pools defined?  we should clarify SOMp, SOMc, and SOMa as som1, som2, som3

Reply:
Ok, done!

@wwieder this comment, together with comments about 2 litter pools not 3, helped me realize that I wasn't thinking of `i_cwd, i_soil1, i_soil2, i_soil3, i_micr1, i_micr2` correctly. I have reworked these and will explain in our meeting.